guides/doc-Planning_for_Project/topics/Hosts.adoc

```diff
@@ -22,17 +22,16 @@ Locations can be used to map the network infrastructure to prevent incorrect hos
 For example, you cannot assign a subnet, domain, or compute resources directly to a {SmartProxyServer}, only to a location.
 
 * *Host groups* – the main carriers of host definitions including assigned Puppet classes, Content View, or operating system.
-ifndef::orcharhino[]
-Find the complete list of host group parameters in {MultiBaseURL}puppet_guide/chap-red_hat_satellite-puppet_guide-adding_puppet_modules_to_red_hat_satellite_6#sect-{Project_Link}-Puppet_Guide-Adding_Puppet_Modules_to_{Project_Link}_6-Parameters[Parameters] in the _Puppet Guide_.
-endif::[]
 It is recommended to configure the majority of settings at the host group level instead of defining hosts directly.
 Configuring a new host then largely becomes a matter of adding it to the right host group.
 As host groups can be nested, you can create a structure that best fits your requirements (see xref:sect-Architecture_Guide-Host_Group_Hierarchies[]).
 
 * *Host collections* – a host registered to {ProjectServer} for the purpose of subscription and content management is called *content host*.
 Content hosts can be organized into host collections, which enables performing bulk actions such as package management or errata installation.
 
-Locations and host groups can be nested, organizations and host collections are flat.
+Locations and host groups can be nested.
+Organizations and host collections are flat.
```

Comment:
Note orgs can be nested in vanilla Foreman. Katello disables that. It also introduces the concept of host collections, which vanilla Foreman doesn't have.
```suggestion
ifdef::katello,satellite,orcharhino[]
Organizations and host collections are flat.
endif::[]
```